plugins/catalog-backend-module-incremental-ingestion/src/engine/IncrementalIngestionEngine.ts

```diff
@@ -329,4 +334,69 @@ export class IncrementalIngestionEngine implements IterationEngine {
       removed,
     });
   }
+
+  async onEvent(params: EventParams): Promise<void> {
+    const { topic, eventPayload } = params;
+    if (topic !== this.providerEventTopic) {
+      return;
+    }
+
+    const { logger, provider, connection } = this.options;
+    const providerName = provider.getProviderName();
+    logger.info(
+      `incremental-engine: Received ${this.providerEventTopic} event`,
```

Comment:
if you had the full EventParams as suggested, you'd get the topic from there instead

Reply:
Also should this be on `info` level? Might be sensible to lower to debug, otherwise it'll get pretty noisy on busy topics

Reply:
Switched to debug. Good call.